src/redux/team-members/selectors.js

```diff
@@ -22,6 +22,17 @@ export const teamMembersErrorSelector = createSelector(
 
 export const teamMembersByFieldSelector = createSelector(
   teamMembersRootSelector,
-  ({ items, sortByField }) =>
-    [...items].sort((a, b) => (a[sortByField] > b[sortByField] ? 1 : -1))
+  ({ items, sortByField, sortDirection }) =>
+    [...items].sort((a, b) => {
+      if (sortDirection === true) {
```

Comment:
Is the `=== true` really necessary?

It's not necessarily better, but you could also put that condition in the ternary:
`a[sortByField] > b[sortByField] ? (sortDirection ? 1 : -1) : (sortDirection ? -1 : 1);`
That way you can lose the `return` and return the expression immediately.